### PR TITLE
Done company

### PR DESCRIPTION
**Pull request by Bountyhunter159**

### CODE REVIEW

File: collections/src/main/java/library/book/Main.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
+package library.book;
+
+
+import java.sql.SQLOutput;
```
Comment (nmelnyk-corevalue) [inline collections/src/main/java/library/book/Main.java]: ?

File: company/src/main/java/WorkWithStuff.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+import java.util.*;
+
+public class WorkWithStuff {
+    public List getStuff() {
+        Stuff stuff[] = Stuff.values();
+        List<Emploee> employees = new LinkedList();
+        Random number = new Random();
+        Faker faker = new Faker();
+        Emploee emploee;
+        for (int i = 0; i < 5; i++) {
```
Comment (nmelnyk-corevalue) [inline company/src/main/java/WorkWithStuff.java]: can we encapsulate that big chunk?

File: company/src/main/java/WorkWithStuff.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+            }
+        }
+        return employees;
+    }
+
+    public List filterBySalary(List stuff, double salary) {
+       List <Emploee>employees = stuff;
+       Emploee emploee;
+        for (int i = 0; i < employees.size() ; i++) {
+            emploee = (Emploee) employees.get( i );
```
Comment (nmelnyk-corevalue) [inline company/src/main/java/WorkWithStuff.java]: casting?

**Comment by nmelnyk-corevalue** [comment]
https://prnt.sc/pyilkz
put that in the right place pliz

File: collections/src/main/java/library/book/Main.java (hunk 1 above)
Comment (Bountyhunter159) [reply collections/src/main/java/library/book/Main.java]: Fixed

File: company/src/main/java/WorkWithStuff.java (hunk 2 above)
Comment (Bountyhunter159) [reply company/src/main/java/WorkWithStuff.java]: I make all methods default

File: company/src/main/java/WorkWithStuff.java (hunk 3 above)
Comment (Bountyhunter159) [reply company/src/main/java/WorkWithStuff.java]: emploee = (Emploee) employees.get( i );  It means I must write as
 emploee =  employees.get( i );

Comment (nmelnyk-corevalue) [reply company/src/main/java/WorkWithStuff.java]: sure

**Pull request closed without merging**